strapi/strapi
# fix: The list view settings popover is not scrollable

I limited the maximum height and added the ability to scroll content inside the block.

### What does it do?

Describe the technical changes you did.
Added <code>maxHeight={'240px'}</code> and <code>overflow={'scroll'}</code> properties for parent <code>Flex</code> component in <code>ViewSettingsMenu</code>.

### Why is it needed?

Scrolling of the content becomes possible when the block is filled with a large amount of content.

### How to test it?

- Go to the list view of a content-type with many different fields
- Click on the settings of the list view
- See the long popover and the ability to scroll

### Related issue(s)/PR(s)

Closes #21572
